java/ddlogapi.c

```diff
@@ -220,6 +220,52 @@ JNIEXPORT jint JNICALL Java_ddlogapi_DDlogAPI_ddlog_1transaction_1commit_1dump_1
     return (jint)result;
 }
 
+JNIEXPORT jint JNICALL Java_ddlogapi_DDlogAPI_ddlog_1transaction_1commit_1dump_1changes_1to_1flatbuf(
+    JNIEnv * env, jobject obj, jlong handle, jobject fbdescr)
+{
```

Comment:
the rest of the file is indented with the curly brace at the end of the previous line